__tests__/app.test.js

```diff
@@ -56,4 +56,46 @@ describe("/api", () => {
             expect(body).toEqual(endpoints)
         })
     })
+})
+
+describe("/api/articles/:article_id", () => {
+    test("GET 200: Should return an object", () => {
```

Comment:
This check for the body to be an object is implicitly tested in your next test, so not necessary to include here. Nice to see as proof of you considering your TDD approach though 😄